app/views/posts/index.html.erb

```diff
@@ -1 +1,29 @@
-<h1>Great Quotes</h1>
\ No newline at end of file
+<h1>Great Quote</h2>
+<div class="container">
+  <div class="user_info">
+    <div class="img">
+      <%= image_tag @user.photo, alt: "User Photo", class: "user-photo" %>
+    </div>
+    <div class="user">
+      <h2><%= link_to @user.name, user_path(@user) %></h2>
+      <p><strong>Number of Posts:</strong> <%= @user.posts.count %></p>
+    </div>
+  </div>
+
```

Comment:
- [ ] Fantastic effort so far! 👏 However, I've come across an error when clicking the See all posts button 🐛. Please make sure to fix this issue to ensure a smoother user experience.

_Due to this bug, I am unable to navigate through the post index page. Please ensure that it is functioning correctly so that the next reviewer can assess it_

![image](https://github.com/evansnyamekye/blog-app/assets/106119105/367cc1a4-c079-4131-97af-8ed5030e07c4)